lib/saml_idp/request.rb

```diff
@@ -105,7 +105,7 @@ def requested_aal_authn_context
     def requested_vtr_authn_context
```

Comment:
should we make this a pluralized method? since we're charing the type?
```suggestion
    def requested_vtr_authn_contexts
```